pkg/datasource.go

```diff
@@ -663,3 +780,83 @@ func (o *OCIDatasource) generateCustomMetricLabel(legendFormat string, metricNam
 		"metricName", metricName, "metricLabel", metricLabel)
 	return metricLabel
 }
+
+func (o *OCIDatasource) tenancyConfigResponse(ctx context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
+	resp := backend.NewQueryDataResponse()
+	var oci_config_file string
+
+	for _, query := range req.Queries {
+
+		if _, ok := os.LookupEnv("OCI_CONFIG_FILE"); ok {
+			oci_config_file = os.Getenv("OCI_CONFIG_FILE")
+		} else {
+			oci_config_file = "/home/grafana/.oci/config"
+		}
+
+		file, err := os.Open(oci_config_file)
```

Comment:
I need to read the file because it is not just used for the configFileProvider call, but also to populate the "config" dropdown list in the plugin query.